README.md

```diff
@@ -51,6 +51,8 @@ Name     | Description
 ---------|------------
 startDelaySeconds | start delay before serving requests. Any requests within the delay period will result in an empty metrics set.
 hostPort | The host and port to connect to via remote JMX. If neither this nor jmxUrl is specified, will talk to the local JVM.
+username | The username to be used in JMX password authentication.
```

Comment:
Done. Would you like me to squash the commits? (If yes, by force pushing or new squashed PR?).